### DISCUSSION TITLE
CSS file is not working on GitHub

### DISCUSSION THREAD

**Discussion by samzea** [community/community#28678] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello,

My CSS file is not working on GitHub. Only html works. I checked all the syntax rules, spelling, etc. I went to validator site. I also removed the cache on my browser. But nothing works.

This is my site  https://ybigford.github.io/portfo/

And my code

  
    <link rel=“stylesheet” href=“styles.css”>
   

Could anybody help me please?

**Comment by airtower-luna** [comment, marked as answer]
Your code is using paired quotation marks instead of double quotes (`"`), which makes it broken HTML. Attributes must be enclosed in simple double quotes. Browsers tend to try to fix broken HTML, my Firefox tries to make sense of it by adding the missing double quotes:

```html
<link rel="“stylesheet”" href="“styles.css”">
```

But `“stylesheet”` isn't a defined `rel`, and even if it was there's no file called `“styles.css”` (including the quotation marks!) in your repository.

Use simple double quotes for all the HTML attributes, and it should work:

```html
<link rel="stylesheet" href="styles.css">
```

Note that a bunch of other elements in your HTML `<head>` have the same issue.

**Reply by samzea** [reply]
I fixed it and it works! Thanks SOOOOOO MUCH!!!!! 👍👍👍👍👍😀

**Reply by airtower-luna** [reply]
Excellent, I'm glad that did it! :smile_cat: 

Marking the answer would be appreciated.
